docs/admin-ports.md

```diff
@@ -38,6 +38,8 @@ The following is a list of ports used by internal DC/OS services, and their corr
  - 8181: dcos-exhibitor
  - 9990: dcos-cosmos
  - 15055: dcos-history-service
+ - 15101: dcos-marathon libprocess
+ - 15201: dcos-metronome libprocess
```

Comment:
Marathon uses 8080 (http) and 8443 (https) by default. For metronome these are 9090 and 9443. I thought giving libmesos it's "own range" so it's distinct from others.